pkg/exchange/okex/exchange.go

```diff
@@ -339,3 +339,30 @@ func (e *Exchange) QueryKLines(ctx context.Context, symbol string, interval type
 	return klines, nil
 
 }
+
+func (e *Exchange) QueryOrder(ctx context.Context, q types.OrderQuery) (*types.Order, error) {
+	if len(q.Symbol) == 0 {
+		return nil, errors.New("okex.QueryOrder: InstrumentID is required parameter")
+	}
+	if len(q.OrderID) == 0 && len(q.ClientOrderID) == 0 {
+		return nil, errors.New("okex.QueryOrder: ordId or clOrdId is required parameter")
```

Comment:
The error message is for human, and `clOrdId` is not cleared enough. Using `ClientOrderID` directly is clearer.

Reply:
Updated.